tests/L0/run_fused_layer_norm/test_fused_layer_norm.py

```diff
@@ -4,38 +4,40 @@
 
 import torch
 import apex
+from torch.autograd import Variable
 
         
 class TestFusedLayerNorm(unittest.TestCase):
     def setUp(self):
-        self.module = apex.normalization.FusedLayerNorm(normalized_shape=[32, 64], elementwise_affine=False)
-        self.input_ = torch.randn(16, 32, 64)
+        self.module_cpu_ = apex.normalization.FusedLayerNorm(normalized_shape=[32, 64], elementwise_affine=False).cpu()
+        self.module_cuda_ = apex.normalization.FusedLayerNorm(normalized_shape=[32, 64], elementwise_affine=False).cuda()
         torch.cuda.manual_seed(42)
-        
-    def forward_cpu(self, input_):
-        self.module.cpu()
-        return self.module(input_.cpu())
-    
-    def forward_cuda(self, input_):
-        self.module.cuda()
-        return self.module(input_.cuda())
-    
-    def test_forward_cuda(self):
-        out_ = self.forward_cuda(self.input_)
-        assert out_.is_cuda == True
-        
-    def test_forward_cpu(self):
-        out_ = self.forward_cpu(self.input_)
-        assert out_.is_cuda == False
-        
+        self.input_ = torch.randn(16, 32, 64)
+        self.loss_ = torch.randn(16, 32, 64)
+        self.input_cpu_ = Variable(self.input_.detach().clone(), requires_grad=True)
```

Comment:
`Variable` is an old syntax that's going to be deprecated, can you please replace it with
```
self.input_cpu_ = self.input_.clone().detach().requires_grad_(True)
```
similar for input_cuda_ below.